scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/RecurrenceGoal.java

```diff
@@ -173,31 +154,31 @@ protected RecurrenceGoal() { }
       Duration start, Duration end, Plan plan)
   {
     final var conflicts = new java.util.LinkedList<MissingActivityConflict>();
-
-    var goalConstraint = this.stateConstraints;
-
     //determine how much flexibility there is in creating activities
     final var recurrenceFlexibility = recurrenceInterval.end.minus(
         recurrenceInterval.start);
 
+    if(end.minus(start).noLongerThan(recurrenceInterval.end)){
```

Comment:
`noLongerThan` is kinda weird. Would this work better `notLongerThan`?

Reply:
You want to change the `Duration` method names everywhere?

Reply:
The methods were named in the pattern "x is shorter than y" (strictly less than) and "x is no longer than y" (less-than-or-equal). While "x is not longer than y" technically says the same thing, it doesn't sound right to my ear.